styleguide/markdown-syntax-style.md

```diff
@@ -1,157 +1,88 @@
-
-_"The quick brown fox jumps over the lazy dog."_
-
-The standard case used in English prose. That is, only the first word is capitalized, except for proper nouns and other words which are generally capitalized by a more specific rule.
+### Headings
 
-### Capital letters in proper names
+Ensure headings are always nested incrementally within any Markdown page (that is, `# Page title`, `## Heading`,`### Heading`, `#### Heading`, etc.) throughout the docs, where the first heading level is the page title. Be aware that this incremental nesting rule can be broken on the way up. For example:
 
-Only capitalize the name of the product - for example, 'GitHub organization' and not 'GitHub Organization'.
+```
+# Page title
 
-### Capital letters in UX elements
+Some text.
 
-Match the capitalization used in the Buildkite interface.
+## Heading level 1
 
-### Lists (bullet lists and numbered steps)
+More text.
 
-Capitalize the first word; no full stops at the end if it's not a full sentence. If it's a full sentence, give it a full stop.
+### Heading level 2
 
-See also what Microsoft has to say on [lists](https://docs.microsoft.com/en-us/style-guide/scannable-content/lists).
+Even more text.
 
-### Writing numbers
+#### Heading level 3
 
-Write out numbers up to 10, then use digits - '58 bugs in this script and just two hours to fix them all!'
-Long numbers use commas to separate thousands - '100,000,000.00'. When in doubt, look [here](https://docs.microsoft.com/en-us/style-guide/numbers).
+...
 
-Do not use numbers in page headings. Only use numbers less than 10 in section headings.
-No restrictions on using numbers in the body of the text.
+#### Another heading level 3
 
+...
 
-### Referring to UI elements
+## Another heading 1
 
-UI elements should be italic. For example:
+...
+```
 
-To get your agent token, navigate to _Agents_, then select _Reveal Agent Token_.
+> [!NOTE]
+> To improve the readability of the Markdown source content, ensure there is an empty line inserted both above and below the heading.
 
-> **Note:** Markdown supports two characters as its markup for itaclising text - either an underscore "\_" or a single asterisk "\*". For consistency, use single underscores "\_" immediately surrounding the text you want to italicise - e.g. `_Italicise this text_`
+Refer to [Heading capitalization in the Writing style guide](writing-style.md#heading-capitalization) for details on how to write and present headings in the Buildkite docs.
 
-### Time and date
+### UI elements
 
-Use 24hr time with hours and minutes, but not seconds. Include timezone. For example, 17:00 AEST
+UI element references are formatted using italics in the Buildkite docs. Markdown supports two characters as its markup for italicizing text - either an underscore "\_" or a single asterisk "\*". For consistency, use single underscores "\_" immediately surrounding the text you want to italicize - e.g. `_Italicize this text_`
```

Comment:
Interesting! I like the idea of using an em dash for this purpose but wasn't aware of the lack of the 'no spacing' around it. I was a little taken aback by this but I don't have strong opinions about it, and I'm soon getting used to it :-)
Since we've decided on the Google develop documentation style guide as our main set of guidelines. I'm happy to go along with this.
I've also now included the references to these Google style guide (+ Microsoft Style Guide) at the top of the Markdown syntax style guide.